features/react_native.feature

```diff
@@ -11,3 +11,80 @@ Scenario: React Native sends requests
     And 1 requests are valid for the android mapping API and match the following:
       | versionCode | versionName | appId                     |
       | 1           | 1.0         | com.bugsnag.android.rnapp |
+
+    And 1 requests are valid for the JS source map API and match the following:
+        | appVersionCode | appVersion | platform |
+        | 1              | 1.0        | android  |
+
+Scenario: Source maps are uploaded when bundling an app with the default project structure
```

Comment:
This is identical in steps to the first currently.

Reply:
Updated to invoke the bundle command as a script instead.